### ISSUE TITLE
1:train loss decrease too faster.2:learning rates did not change after warmup iter,always kept on 1.5e-4. Is it normal phenomenon? 

### ISSUE THREAD

**Issue by kangzhonghua** [NVIDIA/Megatron-LM#14]
Thank your very much! I was encounter some question.
     1:train loss decrease too faster.
     2:learning rates did not change after warmup iter,always kept on 1.5e-4. 
Is it normal phenomenon?

total 10GB chinese corpus ,about 3000000 sample.
python3 -m torch.distributed.launch \
    --nnodes 1 \
    --nproc_per_node 2 \
    pretrain_gpt2.py \
        --num-layers 24 \
        --hidden-size 1024 \
        --num-attention-heads 16 \
        --max-position-embeddings 1024 \
        --seq-length 1024 \
        --batch-size 8 \
        --train-iters 1000000 \
        --save-interval 1000 \
        --save checkpoints/gpt2_345m_hm10g \
        --load checkpoints/gpt2_345m_hm10g \
        --tensorboard-dir logs/gpt2_345m_hm10g \
        --resume-dataloader \
        --train-data corpus_data \
        --lazy-loader \
        --tokenizer-type SentencePieceTokenizer \
        --tokenizer-path data/spm/corpus_bpe_32k.model \
        --cache-dir cache \
        --split 949,50,1 \
        --distributed-backend nccl \
        --lr 0.00015 \
        --lr-decay-style cosine \
        --weight-decay 1e-2 \
        --clip-grad 1.0 \
        --warmup .01 \
        --checkpoint-activations \
        --fp16 \

![a_t](https://user-images.githubusercontent.com/12265127/66694293-9d768680-ece4-11e9-9ab8-cca43588c41e.png)
![b_g_t](https://user-images.githubusercontent.com/12265127/66694294-9d768680-ece4-11e9-8d27-ff3ed20f5aeb.png)
![b_t](https://user-images.githubusercontent.com/12265127/66694295-9ea7b380-ece4-11e9-89d1-7fc1fdce9db1.png)
![d_l_t](https://user-images.githubusercontent.com/12265127/66694296-9ea7b380-ece4-11e9-8a1e-38dd269886e8.png)
![f_t](https://user-images.githubusercontent.com/12265127/66694299-9f404a00-ece4-11e9-80c1-2d04108bf28f.png)
![i_t](https://user-images.githubusercontent.com/12265127/66694300-9f404a00-ece4-11e9-8dfd-950a771741f9.png)
![l_s](https://user-images.githubusercontent.com/12265127/66694301-9fd8e080-ece4-11e9-94db-0d2fa06893ae.png)
![lr](https://user-images.githubusercontent.com/12265127/66694302-9fd8e080-ece4-11e9-8f0d-33b142c88579.png)
![o_t](https://user-images.githubusercontent.com/12265127/66694303-a0717700-ece4-11e9-9a5f-2b05d04b717b.png)
![t_l](https://user-images.githubusercontent.com/12265127/66694304-a0717700-ece4-11e9-827b-174bdfc1bad6.png)
![v_l](https://user-images.githubusercontent.com/12265127/66694305-a10a0d80-ece4-11e9-8e99-0f98493ea7f7.png)
![v_ppl](https://user-images.githubusercontent.com/12265127/66694306-a10a0d80-ece4-11e9-9d90-b8aac6632e24.png)

**Comment by jon-barker** [comment]
Closing issue due to age
